src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -176,23 +176,15 @@ private void RequestRefresh(AuthenticationTicket ticket)
 
         private CookieOptions BuildCookieOptions()
         {
-            var cookieOptions = new CookieOptions
-            {
-                Domain = Options.CookieDomain,
-                SameSite = Options.CookieSameSite,
-                HttpOnly = Options.CookieHttpOnly,
-                Path = Options.CookiePath ?? (OriginalPathBase.HasValue ? OriginalPathBase.ToString() : "/"),
-            };
+            var cookieOptions = Options.Cookie.Build(Context);
+            // ignore the 'Expires' value as this will be computed elsewhere
```

Comment:
This is fine, I just wanted to make sure we are using the expires overload somewhere, and it looks like we are with the nonce/correlation cookies so its cool.  Otherwise I'd have suggested removing the unused overload.